cdap-ui/app/cdap/components/DataPrep/TopPanel/PipelineConfigHelper.js

```diff
@@ -336,6 +336,43 @@ function constructBigQuerySource(artifactsList, bigqueryInfo) {
   };
 }
 
+function constructSpannerSource(artifactsList, spannerInfo) {
+  if (!spannerInfo) { return null; }
+
+  let batchArtifact = find(artifactsList, {name: 'google-cloud'});
+  if (!batchArtifact) {
+    return T.translate(`${PREFIX}.spanner`);
+  }
+
+  batchArtifact.version = '[0.9.2, 3.0.0)';
```

Comment:
what is this version range correspond to?
Will we need to be updating it in later versions of CDAP?

Reply:
This corresponds to the version of `google-cloud` plugin when Spanner source is added https://github.com/data-integrations/google-cloud/blob/develop/pom.xml#L22. Have updated it to use the right value.

We'll only need to update it if the version of `google-cloud` plugin is >= 3.0.0.